runtime/gc_vlhgc/CopyForwardScheme.cpp

```diff
@@ -2906,7 +2906,7 @@ MM_CopyForwardScheme::bestCacheForScanning(MM_CopyScanCacheVLHGC* copyCache, MM_
 	if (!copyCache->isScanWorkAvailable()) {
 		return false;
 	}
-	if (!((*scanCache)->flags & J9VM_MODRON_SCAVENGER_CACHE_TYPE_COPY)) {
+	if (!((*scanCache)->flags & OMR_SCAVENGER_CACHE_TYPE_COPY)) {
```

Comment:
this is probably not a style that conforms with coding standard.
we should compare it with 0 (which Scavenger.cpp) uses, or even use some of bitwise macros (common in core VM code, but no so in GC)

Reply:
Tests like this should use `OMR_ARE_ANY_BITS_SET()`.